TinkerWorX.SharpCraft.Launcher/Start game map file.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+Launcher.exe -game -loadfile "L:\Program Files (x86)\Warcraft III\Maps\some-map.w3x" -window
```

Comment:
I think we should remove this. Since the path is machine specific, it's not very portable.